gr-blocks/lib/wavfile_sink_impl.cc

```diff
@@ -86,6 +86,9 @@ wavfile_sink_impl::wavfile_sink_impl(const char* filename,
     set_bits_per_sample_unlocked(bits_per_sample);
     d_h.bytes_per_sample = d_bytes_per_sample_new;
 
+    set_max_noutput_items(ITEMS_SIZE);
+    sample.resize(ITEMS_SIZE * d_h.nchans);
```

Comment:
slight optimization is to init `sample` in initializer list as `d_sample(n_channels)`.

Won't affect runtime performance, only construction. But in my opinion makes it clearer for the reader to see the size.

Reply:
Did you mean ```d_sample(ITEMS_SIZE * n_channels)```?

Reply:
Yes, sorry.

Reply:
I'm going to stick with ```resize()```, just to keep it the same in both files (n_channels is not available in the source block).